cmd/soroban-rpc/lib/preflight/src/fees.rs

```diff
@@ -23,9 +23,9 @@ use state_expiration::{get_restored_ledger_sequence, ExpirableLedgerEntry};
 use std::cmp::max;
 use std::convert::{TryFrom, TryInto};
 
-/// Estimate for any `ExpirationEntry` ledger entry, consisting of a 32-byte
-/// hash of the corresponding entry and 4 bytes for expiration ledger.
-const EXPIRATION_ENTRY_SIZE: u32 = 32 + 4;
+// TODO: this should be imported from soroban_env_host::fees instead
+/// Serialize XDR size for any `ExpirationEntry` ledger entry.
+const EXPIRATION_ENTRY_SIZE: u32 = 48;
```

Comment:
It would be nice to have a unit test that validate that magic number - but not a blocker.

Reply:
That should be in the upstream library, no here (check the TODO)

Reply:
I added a test at https://github.com/stellar/rs-soroban-env/pull/1034